docs/root/start/sandboxes/dynamic-configuration-filesystem.rst

```diff
@@ -0,0 +1,94 @@
+.. _install_sandboxes_dynamic_config_fs:
+
+Dynamic configuration (filesystem)
+==================================
+
+This example walks through configuring Envoy using filesystem-based dynamic configuration.
+
+It demonstrates how configuration provided to Envoy dynamically can be updated without
+restarting the server.
+
+.. include:: _include/docker-env-setup.rst
+
+Change directory to ``examples/dynamic-config-fs`` in the Envoy repository.
+
+Step 3: Start the proxy container
+*********************************
+
+.. note::
+
+   If you are running on a system with strict ``umask`` you will need to ``chmod`` the dynamic config
```

Comment:
im not over-happy with this, but after thinking about it quite a bit, it seems like the least bad option to handle the `umask` issue